tuf_conformance/test_basic.py

```diff
@@ -160,14 +160,18 @@ def test_timestamp_content_changes(
     assert client.init_client(init_data) == 0
     assert client.refresh(init_data) == 0
 
-    initial_timestamp_meta_ver = repo.timestamp.snapshot_meta.version
-    # Change timestamp without bumping its version in order to test if a new
-    # timestamp with the same version will be persisted.
+    # Change timestamp v1: client should not download a new one if it has v1 already
```

Comment:
My comment is incorrect: timestamp is always downloaded, we just expect the client to keep using the version it already has.